File: src/main/java/tech/zolhungaj/amqapi/servercommands/gameroom/lobby/PlayerChangedAvatar.java
package tech.zolhungaj.amqapi.servercommands.gameroom.lobby;

import tech.zolhungaj.amqapi.servercommands.CommandType;
import tech.zolhungaj.amqapi.servercommands.objects.PlayerAvatar;

@CommandType("avatar change")
public record PlayerChangedAvatar(
        int gamePlayerId,
        PlayerAvatar avatar
){}

File: src/main/java/tech/zolhungaj/amqapi/servercommands/gameroom/lobby/PlayerChangedAvatar.kt
package tech.zolhungaj.amqapi.servercommands.gameroom.lobby

import tech.zolhungaj.amqapi.servercommands.CommandType
import tech.zolhungaj.amqapi.servercommands.objects.PlayerAvatar

@JvmRecord
@CommandType("avatar change")
data class PlayerChangedAvatar(
    val gamePlayerId: Int,
    val avatar: PlayerAvatar
)
